newsroom/wire/search.py

```diff
@@ -282,22 +286,17 @@ def has_permissions(self, item, ignore_latest=False):
         except Forbidden:
             return False
 
-    def get_product_items(self, product_id, size):
+    def get_product_items(self, product_id, size, company_id=None):
         query = _items_query()
 
         product = get_resource_service('products').find_one(req=None, _id=product_id)
 
         if not product:
             return
 
-        query['bool']['must'].append({
-            "bool": {
-                "should": [
-                    {"range": {"embargoed": {"lt": "now"}}},
-                    {"bool": {"must_not": {"exists": {"field": "embargoed"}}}}
-                ]
-            }
```

Comment:
so how is this filter applied now?

Reply:
There's a filter in company_types https://github.com/superdesk/newsroom/blob/stt/settings.py#L61
So this one will be used now. But let me know if you think we need another way.

Reply:
I see, np then